src/config.c

```diff
@@ -544,6 +559,36 @@ static char *getRedisConfig(RedisModuleCtx *ctx, const char *name)
     return buf;
 }
 
+void ConfigInit(RedisModuleCtx *ctx, RedisRaftConfig *config)
+{
+    UNUSED(ctx);
+
+    memset(config, 0, sizeof(RedisRaftConfig));
+
+    config->raft_log_filename = RedisModule_Strdup(REDIS_RAFT_DEFAULT_LOG_FILENAME);
+    config->raft_interval = REDIS_RAFT_DEFAULT_INTERVAL;
+    config->request_timeout = REDIS_RAFT_DEFAULT_REQUEST_TIMEOUT;
+    config->election_timeout = REDIS_RAFT_DEFAULT_ELECTION_TIMEOUT;
+    config->connection_timeout = REDIS_RAFT_DEFAULT_CONNECTION_TIMEOUT;
+    config->join_timeout = REDIS_RAFT_DEFAULT_JOIN_TIMEOUT;
+    config->reconnect_interval = REDIS_RAFT_DEFAULT_RECONNECT_INTERVAL;
+    config->raft_response_timeout = REDIS_RAFT_DEFAULT_RAFT_RESPONSE_TIMEOUT;
+    config->proxy_response_timeout = REDIS_RAFT_DEFAULT_PROXY_RESPONSE_TIMEOUT;
+    config->raft_log_max_cache_size = REDIS_RAFT_DEFAULT_LOG_MAX_CACHE_SIZE;
+    config->raft_log_max_file_size = REDIS_RAFT_DEFAULT_LOG_MAX_FILE_SIZE;
+    config->raft_log_fsync = true;
+    config->quorum_reads = true;
+    config->sharding = false;
+    config->external_sharding = false;
+    config->slot_config = "0:16383";
+    config->shardgroup_update_interval = REDIS_RAFT_DEFAULT_SHARDGROUP_UPDATE_INTERVAL;
+    config->max_appendentries_inflight = REDIS_RAFT_DEFAULT_MAX_APPENDENTRIES;
+    config->tls_ca_cert = getRedisConfig(ctx, "tls-ca-cert-file");
+    config->tls_key = getRedisConfig(ctx, "tls-key-file");
+    config->tls_cert = getRedisConfig(ctx, "tls-cert-file");
```

Comment:
Redis supports `tls-client-cert-file` and `tls-client-key-file` to allow using different certificates on the client side and on the server side. I think this should also be supported.

Also, `tls-key-file-pass` and `tls-client-key-file-pass` should be supported.

Reply:
ok, can add those in a future PR.  will create a new ticket